src/Library/graphql.ts

```diff
@@ -1,5 +1,29 @@
 import gql from 'graphql-tag'
 
+export type SongQuery = {
+  data: {
+    song: Song
+  }
+  vars: {
+    id: string
+  }
+}
+
+export const SONG_QUERY = gql`
+  query Song($id: ID!) {
+    song(id: $id) {
+      id
+      durationInSeconds
```

Comment:
Nice! Does this mean we can now access the duration of a song in other places throughout the application?